contracts/bindings/scripts/gen.sh

```diff
@@ -1,21 +1,26 @@
 #!/usr/bin/env bash
-# Generate bindings for the given solidity contracts, to be
+# Generate bindings for solidity contracts
 
 DIR=${DIR:-./bindings}
 PKG=${PKG:-bindings}
 
 # generate bindings for the given contract
+# works on contract name of fully qualified path to the contract
 # params:
-#  $1: contract name (ex. 'OmniPortal')
+#  $1: contract name (ex. 'OmniPortal', 'src/protocol/OmniPortal.sol:OmniPortal')
 gen_binding() {
-  name=$1
+  contract=$1
+
+  # strip path prefix, if used
+  # ex src/protocol/OmniPortal.sol:OmniPortal => OmniPortal
+  name=$(echo ${contract} | cut -d ":" -f 2)
```

Comment:
Ran into issues with `forge inspect MockERC20 abi` returning the wrong abit - so needed to use the full path